core/server/worker/src/main/java/alluxio/worker/block/SpaceReserver.java

```diff
@@ -59,25 +61,57 @@ public SpaceReserver(BlockWorker blockWorker) {
     Map<String, Long> capOnTiers = blockWorker.getStoreMeta().getCapacityBytesOnTiers();
     long lastTierReservedBytes = 0;
     for (int ordinal = 0; ordinal < mStorageTierAssoc.size(); ordinal++) {
-      PropertyKey tierReservedSpaceProp =
-          PropertyKeyFormat.WORKER_TIERED_STORE_LEVEL_RESERVED_RATIO_FORMAT.format(ordinal);
       String tierAlias = mStorageTierAssoc.getAlias(ordinal);
-      long reservedSpaceBytes =
-          (long) (capOnTiers.get(tierAlias) * Configuration.getDouble(tierReservedSpaceProp));
-      mBytesToReserveOnTiers.put(tierAlias, reservedSpaceBytes + lastTierReservedBytes);
-      lastTierReservedBytes += reservedSpaceBytes;
+      long capOnTier = capOnTiers.get(tierAlias);
+      long reservedBytes;
+      try {
+        PropertyKey tierReservedSpaceProp =
+            PropertyKeyFormat.WORKER_TIERED_STORE_LEVEL_RESERVED_RATIO_FORMAT.format(ordinal);
+        reservedBytes =
+            (long) (capOnTiers.get(tierAlias) * Configuration.getDouble(tierReservedSpaceProp));
+      } catch (RuntimeException e) {
```

Comment:
Instead of catching runtime exception, could we use `Configuration.containsKey` to check the existence of the old property instead?